website/docs/tutorials/structured_config/5_schema.md

```diff
@@ -2,11 +2,11 @@
 id: schema
 title: Structured config schema
 ---
-We have seen how to use Structured Configs as configuration, but they can also be used as a schema validating configuration files!  
-When Hydra loads a configuration, it looks for config with the same name in the `ConfigStore`.
+We have seen how to use Structured Configs as configuration, but they can also be used as schema validating configuration files!
```

Comment:
disagreeing, the a makes sense to me.

Reply:
"a" is singular so doesn't play well with "files" IMO -- I actually just asked an American colleague who confirmed (also suggested to rpelace "as configuration" by "for configuration", so I'll add another commit about that)

Reply:
Scratch that, I think I just understood the meaning of that sentence... will rephrase in another commit as it's ambiguous

Reply:
Let me know if a732a1bcb476e3fdd547cf82e9c7a277d2270089 works for you